Bender.yml

```diff
@@ -35,7 +35,7 @@ dependencies:
   redundancy_cells:       { git: "https://github.com/pulp-platform/redundancy_cells.git", rev: c37bdb47339bf70e8323de8df14ea8bbeafb6583 } # branch: astral_rebase
   redmule:                { git: "https://github.com/pulp-platform/redmule.git", rev: 7abb256 } # branch: fc/hci-v2
   neureka:                { git: "https://github.com/pulp-platform/neureka.git", rev: 60c51c0 } # branch: hci-v2
-  ez_sfm:                 { git: "https://github.com/belanoa/ez_sfm.git"       , rev: 7630cbe } # branch: hci-v2
+  softex:                 { git: "https://github.com/belanoa/softex.git"       , rev: 9da2870 }
```

Comment:
Is this the master branch? Could you please add as a comment the name of the branch it comes from?